### PR TITLE
Remove old toolbar breadcrumbs

### PR DESCRIPTION
**Pull request by juliawegmayr**
## Description

Remove old `ToolbarBreadCrumbs` and replace them with the new ones.

## Example

-   [x] I have verified if my change requires an example

## Changeset

-   [x] I have verified if my change requires a changeset

## Related tasks and documents

https://vivid-planet.atlassian.net/browse/COM-630

### CODE REVIEW

**Review by jamesricky — CHANGES_REQUESTED** [review]
Please add a changeset for this breaking change. 
Even if the usage doesn't change in most use-cases, the props and class-keys have changed which could break existing usages or custom styling.

**Comment by juliawegmayr** [comment]
> Please add a changeset for this breaking change. Even if the usage doesn't change in most use-cases, the props and class-keys have changed which could break existing usages or custom styling.

Thanks for pointing this out. I thought, that the usage would stay the same. Do I need to include more information on how to use the new ones...? d23646eea4ca4c4b013f6bcc98adf3847e4153b2

**Comment by jamesricky** [comment]
> > Please add a changeset for this breaking change. Even if the usage doesn't change in most use-cases, the props and class-keys have changed which could break existing usages or custom styling.
> 
> Thanks for pointing this out. I thought, that the usage would stay the same. Do I need to include more information on how to use the new ones...? [d23646e](https://github.com/vivid-planet/comet/commit/d23646eea4ca4c4b013f6bcc98adf3847e4153b2)

I'm not actually too sure, what the difference is 😅 

If the usage and use-cases of the new component is very similar then it's probably enough to just mention that the props and class-keys have changed. 

If there is actually a change in how certain features of the component are used, you could add a before and after code example to the changeset.

File: .changeset/rich-spiders-trade.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": major
+---
+
+Remove old `ToolbarBreadcrumbs`, `ToolbarBreadcrumbsClassKey` and `ToolbarBreadcrumbsProps`. Use new ones instead.
```
Comment (jamesricky) [inline .changeset/rich-spiders-trade.md:5]: To the users of comet, nothing really is removed, they just see that `ToolbarBreadcrumbs` now looks differently and may have different props/class-keys. 

```suggestion
Redesign the `ToolbarBreadcrumbs` component

Due to internal changes, including the props and class keys, custom usages and styling may need to be adjusted. 
```

**Review by jamesricky — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**